src/main.go

```diff
@@ -30,6 +30,28 @@ func main() {
 		c.JSON(http.StatusOK, gin.H{"id": intID})
 	})
 
+	//　練習課題2.1
+	router.POST("/", func(c *gin.Context) {
+		var json struct {
+			Message string "json:\"message\""
+		}
```

Comment:
バックスラッシュはエスケープシーケンス（次の文字の意味をなくす指令）としてよく用いられる語です。
Message string "json:\"message\""とした場合：{"message": }というダブルクォーテーションを含んだkeyを探す
Message string "json:message"とした場合：{message:}というkeyを探す
ということになります
今回は2個目が正しいと思うので、以下のように修正するのはどうでしょうか
```suggestion
		var json struct {
			Message string "json:message"
		}
```